examples/test_error.py

```diff
@@ -101,17 +101,14 @@
     # }}}
 
     # {{{Normalization
-    frq_slice = integrate_limits(select_pathway(s, signal_pathway),
-            convolve_method='Gaussian')
+    frq_slice = integrate_limits(select_pathway(s, signal_pathway))
     s_integral = s["t2":frq_slice].C # the "official" copy of the integral
     s_integral = select_pathway(s_integral, signal_pathway)
     s_integral.integrate("t2")
-    avg_d = s_integral.C.mean().item()
+    avg_d = s_integral.C.mean().real.item()
     s_integral /= avg_d
     # the line below changes the frequency slice for some reason
     s /= avg_d
-    frq_slice = integrate_limits(select_pathway(s, signal_pathway),
-            convolve_method='Gaussian')
```

Comment:
b/c you were dividing by a complex number (I remove the comment in subsequent commit)